backend/selenium_tests/page_object/programme_population/households.py

```diff
@@ -0,0 +1,125 @@
+from page_object.base_components import BaseComponents
+from selenium.webdriver.remote.webelement import WebElement
+
+
+class Households(BaseComponents):
+    mainContent = 'div[data-cy="main-content"]'
```

Comment:
Many of these attributes and methods can be separated into more general class. Then that class can be used for inheritance